org.cidercollective.cider.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <content_rating type="oars-1.0" />
+  <id>org.cidercollective.cider</id>
```

Comment:
`cidercollective.org` doesn't exist, which make this app-id invalid.

see https://github.com/flathub/flathub/wiki/App-Requirements#application-id

Reply:
Their domain is https://cider.sh/ but they're identifier was https://github.com/ciderapp/Cider/blob/main/flatpak/org.cidercollective.cider.yml so I kept it like that would it required to change it? If so can I do that now or would I need a new PR?

Reply:
you don't need a new PR.

Yes, it should  be changed.

Reply:
Ok it's been done, please verify all occurrences have been updated.